core-bundle/src/Controller/ContentElement/HyperlinkController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Controller\ContentElement;
+
+use Contao\ContentModel;
+use Contao\CoreBundle\Image\Studio\Studio;
+use Contao\CoreBundle\InsertTag\InsertTagParser;
+use Contao\CoreBundle\ServiceAnnotation\ContentElement;
+use Contao\CoreBundle\String\HtmlAttributes;
+use Contao\CoreBundle\Twig\FragmentTemplate;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @ContentElement(category="links")
+ */
+class HyperlinkController extends AbstractContentElementController
+{
+    public function __construct(private readonly Studio $studio, private readonly InsertTagParser $insertTagParser)
+    {
+    }
+
+    protected function getResponse(FragmentTemplate $template, ContentModel $model, Request $request): Response
+    {
+        // Link with attributes
+        $href = $this->insertTagParser->replace($model->url);
```

Comment:
```suggestion
        $href = $this->insertTagParser->replaceInline($model->url);
```

We don’t want to allow ESI tags here I think ☺️